addons/binding/org.openhab.binding.mihome/ESH-INF/thing/aqara_ctrl_ln2.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="mihome" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="ctrl_ln2">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="bridge" />
+		</supported-bridge-type-refs>
+
+		<label>Xiaomi "zero-fire" 2 Channel Wall Switch</label>
+		<description>New, yet unconfirmed device</description>
+
+		<channels>
+			<channel id="ch1" typeId="sw1-btn" />
+			<channel id="ch2" typeId="sw2-btn" />
+		</channels>
+
+		<config-description>
+			<parameter name="itemId" type="text" readOnly="true">
+				<label>Item ID</label>
+				<description>An identifier for the item connected.</description>
+				<required>true</required>
+			</parameter>
+		</config-description>
+	</thing-type>
+
+	<channel-type id="sw1-btn">
+		<item-type>Switch</item-type>
+		<label>Button 1</label>
+		<description>The first pushbutton on the switch.</description>
```

Comment:
oh they toggle?
sorry, I missed that. my imagination told me it has to be a switch 😄 
still they have state because of the relay, where the others trigger click events.